### PR TITLE
Automation scripts improvements

### PR DESCRIPTION
**Pull request by troydai**
Included:
1. The automation package will install these two scripts to the virtual environment's bin folder so that shortcut command `check_style` can be used to run pylint directly.
2. Add flake8 to the automation.style for PEP8 check.

### CODE REVIEW

File: scripts/automation/utilities/path.py [hunk 1]
```diff
@@ -38,8 +38,8 @@ def get_core_modules_paths():
     def _get_path(name):
         return os.path.join(get_repo_root(), 'src', name)
 
-    yield 'azure-cli', _get_path('azure-cli')
-    yield 'azure-cli-core', _get_path('azure-cli-core')
+    yield 'main', _get_path('azure-cli')
+    yield 'core', _get_path('azure-cli-core')
```
Comment (derekbekoe) [inline scripts/automation/utilities/path.py]: Can we keep the original names here?

Comment (derekbekoe) [inline scripts/automation/utilities/path.py]: ...because we also have `azure-cli-nspkg`

Comment (troydai) [inline scripts/automation/utilities/path.py]: ok.

Comment (troydai) [inline scripts/automation/utilities/path.py]: Other than this, good to go?

**Pull request merged**